action.yml

```diff
@@ -58,7 +58,7 @@ outputs:
     description: 'Version of the release built'
 runs:
   using: 'docker'
-  image: 'Dockerfile'
+  image: 'docker://ghcr.io/balena-io/deploy-to-balena-action:latest'
```

Comment:
I think this needs to be versioned with the repo version?

Reply:
Agreed, we should try to get [renovate](https://github.com/balena-io/renovate-config/blob/master/.github/renovate.json#L64) to do it.

Reply:
But if renovate creates a new PR every time a PR merges and the version is bumped we will be stuck in a loop?

The version on this line needs to be bumped to the draft version before or during the merge event.

Reply:
How do other repos using this syntax solve this?

Reply:
I think you can use a custom versionist template/setting thing to make it bump the version in here as well as everywhere else